src/main/kotlin/gg/essential/vigilance/Vigilant.kt

```diff
@@ -407,7 +408,100 @@ abstract class Vigilant @JvmOverloads constructor(
                 placeholder = placeholder,
                 triggerActionOnInitialization = triggerActionOnInitialization,
                 hidden = hidden,
-                action = action
+                action = action,
+            )
+        }
+
+        // Same as [property] but does not need to remain binary-compatible.
```

Comment:
`property` should probably delegate to this one then, so we don't have the same code in two places.